quiche/balsa/balsa_frame_test.cc

```diff
@@ -1709,7 +1709,7 @@ TEST_F(HTTPBalsaFrameTest,
       "trAnsfer-eNcoding:  chunked\r\n"
       "\r\n";
   std::string message_body =
-      "A            chunkjed extension  \r\n"
+      "A             chunked extension  \r\n"
```

Comment:
It's possible that something cares about the precise character stream, but I'm hoping there isn't.